tests/topotests/lib/ospf.py

```diff
@@ -327,18 +326,8 @@ def __create_ospf_global(
                         cmd = "no {}".format(cmd)
                     config_data.append(cmd)
 
-        result = create_common_configuration(
-            tgen, router, config_data, ospf, build, load_config
-        )
-
-    except InvalidCLIError:
-        # Traceback
-        errormsg = traceback.format_exc()
-        logger.error(errormsg)
-        return errormsg
-
-    logger.debug("Exiting lib API: create_ospf_global()")
-    return result
+            logger.debug("Exiting lib API: create_ospf_global()")
```

Comment:
changed indent - would change the debug output?

Reply:
Will fix, now that you've reviewed this, I'll flip the logic on the top-line if and remove an entire level of indent for the entire function.